### ISSUE TITLE
Normalize encoding of binding information

### ISSUE THREAD

**Issue by wlpotter** [srophe/britishLibrary-data#53]
In the rare cases where the `tei:bindingDesc` is not empty, we need to make sure the data is encoded uniformly. I think this is one of the elements where we left "content pending" (on which see #47 ).

**Comment by wlpotter** [comment]
From a quick search, using the below script, I found 11 distinct bindingDesc nodes. One is the 'content pending' one, another is just an empty node. The remaining 9 have real data, so this should be relatively simple editing. Though a quick glance revealed that one of them might not be actual binding data...

```
import module namespace functx="http://www.functx.com";

let $binding-descs :=
  for $doc in collection("path/to/britishLibrary-data/data/tei")
  return $doc//*:bindingDesc
return functx:distinct-deep($binding-descs)
```

**Comment by wlpotter** [comment]
Going to move this up in priority since we will want to handle it in the App display

**Comment by wlpotter** [comment]
These have been fixed separately (see #1520 )
